docs/vNext/Fabric_Configuration_New.md

```diff
@@ -0,0 +1,1115 @@
+> * metadata and private collection support for contracts
+> * support for TLS and mutual TLS communication
+> * option to select the identity for submitting a TX/query
+> * detailed execution data for every TX
+
+## Installing dependencies
+
+You must bind Caliper to a specific Fabric SDK to target the corresponding (or compatible) SUT version. Refer to the [binding documentation](./Installing_Caliper.md#the-bind-command) for details. It is confirmed that a 1.4 Fabric SDK is compatible with a Fabric 2.1 and 2.2 SUT however any installation or instantiation of contracts (chaincode) will only work if you are still using the old lifecycle implementation.
+
+### Binding with Fabric SDK 2.x
+> Note that when using the binding target for the Fabric SDK 2.x there are capability restrictions:
+> * The 2.x SDK does not facilitate administration actions. It it not possible to create/join channels, nor install/instantiate contract. Consequently the 2.x binding only facilitates operation with a `--caliper-flow-only-test` flag
+> * The 2.x SDK only supports operation using a `gateway`. Consequently the 2.x binding requires a `--caliper-fabric-gateway-enabled` flag
+
+### Using the Operational capabilities of the Adapters
+The ability to create channels and perform install/instantiate of contracts (chaincode) remains in the new 1.x sdk based adapters, and nothing equivalent exists for the 2.x sdk based adapters. This capability remains mainly to support the caliper integration tests and the use of these mechanisms to setup your channels and contracts is highly discouraged. This capability is considered legacy and will not be maintained. You should look into alternative methods for setting up your network prior to running caliper to benchmark it.
+
+## Runtime settings
+
+### Common settings
+
+Some runtime properties of the adapter can be set through Caliper's [runtime configuration mechanism](./Runtime_Configuration.md). For the available settings, see the `caliper.fabric` section of the [default configuration file](https://github.com/hyperledger/caliper/blob/master/packages/caliper-core/lib/common/config/default.yaml) and its embedded documentation.
+
+The above settings are processed when starting Caliper. Modifying them during testing will have no effect. However, you can override the default values _before Caliper starts_ from the usual configuration sources.
+
+> __Note:__ An object hierarchy in a configuration file generates a setting entry for every leaf property. Consider the following configuration file:
+> ```yaml
+> caliper:
+>   fabric:
+>     gateway:
+>       enabled: true
+> ```
+> After naming the [project settings](./Runtime_Configuration.md#project-level) file `caliper.yaml` and placing it in the root of your workspace directory, it will override the following two setting keys with the following values:
+> * Setting `caliper-fabric-gateway-enabled` is set to `true`
+>
+> __The other settings remain unchanged.__
```

Comment:
Only shows setting one item, but claims two. Might be a good opportunity to show some other common gateway settings such as `localhost: false`

Reply:
Definitely added in localhost.